Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml

```diff
@@ -13,69 +13,153 @@ References:
-                        EventEndTime = todatetime(TimeGenerated),
-                        EventCount = int(1),
-                        EventVendor = "Microsoft",
-                        EventSchemaVersion = "0.1.0",
-                        EventOriginalType=tostring(EventID),
-                        EventProduct = "Sysmon",
-                        EventResult = 'Success',
-                        DvcOs = "Windows",
-                        ActorUsernameType = "Windows",
-                        ActorUsername = tostring(EventData.User),
-                        TargetProcessName = tostring(EventData.Image),
-                        TargetProcessId = tostring(EventData.ProcessId),
-                        TargetProcessGuid = tostring(EventData.ProcessGuid)
-                | project-rename
-                        DvcHostname = Computer
-                | extend
-                        ActorUsernameType = iff(isnotempty(ActorUsername),'Windows', ''),
-                        //***** Aliases ******
-                        User = ActorUsername,
-                        Process = TargetProcessName,
-                        Dvc = DvcHostname
-                };union isfuzzy=true ParsedProcessEvent_Event,
-                                        ParsedProcessEvent_WindowsEvent   
+    // =======================================================================================================================================
+    //    Sysmon from Event table
+    // =======================================================================================================================================
+    let ParsedProcessEvent_Event=(){
+    Event 
+    | where Source == "Microsoft-Windows-Sysmon" and EventID==5
+    // --------------------------------------------------------------------------------------
+    | where
+    (isnull(starttime)   or TimeGenerated >= starttime )
+    and (isnull(endtime) or TimeGenerated <= endtime )
+    and not(disabled)
+    and (eventtype=='*' or eventtype=='ProcessTerminated')
+    and (array_length(commandline_has_all)==0 ) 
+    and (array_length(commandline_has_any)==0 ) 
+    and (array_length(commandline_has_any_ip_prefix)==0 ) 
+    and (array_length(actingprocess_has_any)==0 ) 
+    and (array_length(targetprocess_has_any)==0 or EventData has_any (targetprocess_has_any)) 
+    and (array_length(parentprocess_has_any)==0 ) 
+    and (targetusername=='*' or EventData has targetusername)  
+    and (array_length(dvcipaddr_has_any_prefix)==0)
+    and (array_length(dvcname_has_any)==0 or Computer has_any (dvcname_has_any))   
+    // --------------------------------------------------------------------------------------
+    | parse EventData with * '<Data Name="User">'ActorUsername '</Data>' *
+    // --------------------------------------------------------------------------------------
+    | where (targetusername=='*' or ActorUsername has targetusername) 
```

Comment:
I think we need to define an actorusername parameter for terminate and not "fake"